<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAQViewer</class>
 <widget class="QWidget" name="DAQViewer">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>302</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>62</red>
        <green>69</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>245</red>
        <green>242</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>125</red>
        <green>165</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>245</red>
        <green>242</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>55</red>
        <green>60</green>
        <blue>61</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>125</red>
        <green>165</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>125</red>
        <green>165</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>125</red>
        <green>165</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>62</red>
        <green>69</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>245</red>
        <green>242</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>125</red>
        <green>165</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>245</red>
        <green>242</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>55</red>
        <green>60</green>
        <blue>61</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>125</red>
        <green>165</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>125</red>
        <green>165</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>125</red>
        <green>165</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>190</red>
        <green>190</green>
        <blue>190</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>62</red>
        <green>69</green>
        <blue>69</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>190</red>
        <green>190</green>
        <blue>190</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>125</red>
        <green>165</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>190</red>
        <green>190</green>
        <blue>190</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>55</red>
        <green>60</green>
        <blue>61</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>55</red>
        <green>60</green>
        <blue>61</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>145</red>
        <green>141</green>
        <blue>126</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Link">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>125</red>
        <green>165</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>51</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>125</red>
        <green>165</green>
        <blue>58</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>XPAD Viewer</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>749</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_ALL">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_Save">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_OpenImage">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Open image</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_SaveAs">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Save image</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_ResetImage">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Reset Image</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_LUT">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox:enabled{
color: rgb(0,0,51);
}
QComboBox:disable{
color: rgb(203, 200, 198);
}</string>
              </property>
              <property name="currentIndex">
               <number>2</number>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>Grayscale</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Hot</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Cold</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Night</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Candy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Geography</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ion</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Thermal</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Polar</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Spectrum</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Jet</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Hues</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="lineEdit_MinValue">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit:enabled{
background: rgb(51,51,51);
color: rgb(252,253,252);
selection-background-color: rgb(126,165,57);
border: rgb(0,0,0);
}
QLineEdit:disable{
background: rgb(51,51,51);
color: rgb(203, 200, 198);
border: rgb(0,0,0);
}</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_Value">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_MaxValue">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit:enabled{
background: rgb(51,51,51);
color: rgb(252,253,252);
selection-background-color: rgb(126,165,57);
border: rgb(0,0,0);
}
QLineEdit:disable{
background: rgb(51,51,51);
color: rgb(203, 200, 198);
border: rgb(0,0,0);
}</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_Acquire">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Acquire</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkBox_Hold">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QCheckBox:enabled{
background: rgb(55, 60, 61);
color: rgb(252,253,252);
}
QCheckBox:disable{
background: rgb(55, 60, 61);
color: rgb(203, 200, 198);
}</string>
              </property>
              <property name="text">
               <string>Hold</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_Continue">
              <property name="styleSheet">
               <string notr="true">QCheckBox:enabled{
background: rgb(55, 60, 61);
color: rgb(252,253,252);
}
QCheckBox:disable{
background: rgb(55, 60, 61);
color: rgb(203, 200, 198);
}</string>
              </property>
              <property name="text">
               <string>Continue</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_ImageNumber">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>190</red>
                <green>190</green>
                <blue>190</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="mode">
           <enum>QLCDNumber::Dec</enum>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="value" stdset="0">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_imageSize">
          <property name="styleSheet">
           <string notr="true">color: white</string>
          </property>
          <property name="text">
           <string>Image size = 0 x 0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_pixelValue">
          <property name="styleSheet">
           <string notr="true">color: white</string>
          </property>
          <property name="text">
           <string>Pixel value = 0, x = 0, y = 0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_customPlot">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QCustomPlot" name="customPlot" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
